src/xrCore/FS.cpp

```diff
@@ -563,6 +568,7 @@ CVirtualFileReader::CVirtualFileReader(const char* cFileName)
 
     data = (char*)MapViewOfFile(hSrcMap, FILE_MAP_READ, 0, 0, 0);
 #elif defined(LINUX)
+    while (char* sep = strchr((char *)cFileName, '\\')) *sep = '/';
```

Comment:
Please consider moving this code (slashes conversion) to separate function (named like ConvertGamePathToPlatformSpecificPath) with platform-specific implementation. It makes the code more clear and allows to reduce code duplication.